php/class-fieldmanager-content.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Abstract class file for Fieldmanager_Content
+ *
+ * @package Fieldmanager
+ */
+
+/**
+ * Static markdown field.
+ */
+abstract class Fieldmanager_Content extends Fieldmanager_Field {
+
+	/**
+	 * Override field_class.
+	 *
+	 * @var string
+	 */
+	public $field_class = 'content';
+
+	/**
+	 * Content to render.
+	 *
+	 * @var string
+	 */
+	public $content = '';
+
+	/**
+	 * Do not save this field. This class is purely informational.
+	 *
+	 * @var bool
+	 */
+	public $skip_save = true;
+
+	/**
+	 * Render content.
+	 *
+	 * @param mixed $value Unused value.
+	 * @return string Rendered content.
+	 */
+	public function form_element( $value = '' ) {
+		return $this->render_content( (string) $this->content );
+	}
+
+	/**
+	 * Render content.
+	 *
+	 * @param string $content Content to render.
+	 * @return string Rendered content.
+	 */
+	abstract protected function render_content( string $content ): string;
```

Comment:
* The implementing classes need to match this signature, so it needs to be public.
* Even though Fieldmanager says it requires PHP 7, we also explicitly say "Fieldmanager is tested down to and works on PHP 5.3-5.6". I'm willing to break with that at some point, but only for a major release, and I don't want to hold this for "2.0".

```suggestion
	abstract public function render_content( $content );
```